resources/htmlTemplate/ExceptionsTocHTMLTemplate.html

```diff
@@ -131,12 +131,12 @@ <h1>License Exceptions</h1>
 
       <ul style="list-style-type: square; padding: 5px;">
-            <li style="margin:5px 0 5px 10px;"><a href="index.html"><strong>SPDX License List</strong></a>: The list of commonly found open source licenses for the purposes of being able to easily and efficiently identify such licenses.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="index.html"><strong>SPDX License List</strong></a>: The list of commonly found free and open source licenses for the purposes of being able to easily and efficiently identify such licenses.</li>
 			<li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML"><strong>Master Files</strong></a>:  The HTML pages you see here are generated from the master files for the SPDX License List.  The master files are XML files describing all the licenses, deprecated licenses, and license exceptions.</li>
             <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-data"><strong>Data Files</strong></a>:  Machine readable files describing all of the licenses and license exceptions.</li>
-            <li style="margin:0px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/license-list-overview"><strong>Overview</strong></a>: For general information about the SPDX License List, including principles for inclusion of a license and an explanation of the fields contained on the list.</li>
+            <li style="margin:0px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/license-list-overview"><strong>Overview</strong></a>: General information about the SPDX License List, including principles for inclusion of a license and an explanation of the fields contained on the list.</li>
             <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>Matching Guidelines</strong></a>:  Guidelines for what constitutes a license match to the SPDX License List. For licenses that include markup, the license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
-            <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/request-new-license"><strong>Request New License</strong></a>: For instructions on how to propose additional licenses or license exceptions be added to the SPDX License List.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/request-new-license"><strong>Request New License</strong></a>: For instructions on how to propose licenses or exceptions be added to the SPDX License List.</li>
```

Comment:
This new line doesn't read quite right to me.  Maybe “be added to” → “for”?

Reply:
I'll accept these for now - we can do another PR after reviewing the updates on preview.